From 0ccadc7a676341bbd66eb176308c7c59931e9dd2 Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@konsulko.com>
Date: Fri, 7 Apr 2023 01:44:56 +0000
Subject: [PATCH] defconfig: switch default kernel compression to LZMA

LZO compression and lzop tool haven't been maintained for long time and have
been removed in recent versions of OE-Core, even though Debian may still use
them. Switch to LZMA compression that is used by default in TI kernel config.

Upstream-Status: Inappropriate [OE-specific]

Signed-off-by: Denys Dmytriyenko <denys@konsulko.com>
---
 arch/arm/configs/bb.org_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/bb.org_defconfig b/arch/arm/configs/bb.org_defconfig
index 6bf0fb860683..7c035735172d 100644
--- a/arch/arm/configs/bb.org_defconfig
+++ b/arch/arm/configs/bb.org_defconfig
@@ -1,5 +1,5 @@
 # CONFIG_LOCALVERSION_AUTO is not set
-CONFIG_KERNEL_LZO=y
+CONFIG_KERNEL_LZMA=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
 CONFIG_NO_HZ_IDLE=y
--
2.25.1
